File: src/breslin/clientside/application/states/applicationGlobal.cpp
//header
#include "applicationGlobal.h"

//log
#include "../../tdreamsock/dreamSockLog.h"

//application
#include "../applicationBreslin.h"

//game
#include "../../game/game.h"

/***************************************
*	CONSTRUCTORS
***************************************/
ApplicationGlobal::ApplicationGlobal(ApplicationBreslin* application)
{
	mApplication = application;
}

ApplicationGlobal::~ApplicationGlobal()
{

}

void ApplicationGlobal::enter()
{

}

void ApplicationGlobal::execute()
{
	if (mApplication->mSetup)
	{
		//check network
		mApplication->checkForByteBuffer();

		//graphics
		mApplication->runGraphics();
	}
}
void ApplicationGlobal::exit()
{
}

File: src/breslin/clientside/application/states/application_global.php
var ApplicationGlobal = new Class(
{
	
initialize: function(application)
{
	this.mApplication = application;
},

enter: function()
{

},

execute: function()
{
	if (this.mApplication.mSetup)
	{
		//graphics
		this.mApplication.runGraphics();
	}
},

exit: function()
{

}

});
